tsconfig.json

```diff
@@ -1,9 +1,40 @@
 {
   "extends": "@tsconfig/node-lts",
   "compilerOptions": {
+    "strict": true,
+
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+    "useUnknownInCatchVariables": true,
+    "strictPropertyInitialization": true,
+    "strictFunctionTypes": true,
+    "noImplicitThis": true,
+    "strictBindCallApply": true,
+    "noPropertyAccessFromIndexSignature": false,
+
+    // disabled to avoid errors in protobuConverter
+    // "exactOptionalPropertyTypes": true,
+    // // Linter Checks
+    "noImplicitReturns": true,
+
+    // this rule broke protobuf-ts
+    // "noImplicitOverride": true,
+    "forceConsistentCasingInFileNames": true,
+    // // https://eslint.org/docs/rules/consistent-return ?
+    "noFallthroughCasesInSwitch": true,
+    // // https://eslint.org/docs/rules/no-fallthrough
     "noUnusedLocals": true,
+    // // https://eslint.org/docs/rules/no-unused-vars
     "noUnusedParameters": true,
-    "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true
+    // // https://eslint.org/docs/rules/no-unused-vars#args
+    "allowUnreachableCode": false,
+    // // https://eslint.org/docs/rules/no-unreachable ?
+    "allowUnusedLabels": false,
+    // // https://eslint.org/docs/rules/no-unused-labels
+    // // Base Strict Checks
+    "noImplicitUseStrict": false,
+    "suppressExcessPropertyErrors": false,
+    "suppressImplicitAnyIndexErrors": false,
```

Comment:
Just curious, why are we setting this to false?

Reply:
I think it makes sense to extend that one!